R/shiny.qmd

```diff
@@ -0,0 +1,67 @@
+# Shiny {.unnumbered}
+
+The easiest way to publish a shiny app is via the Rstudio IDE, either using the desktop version or one of the server based products like Workbench or RStudio Server.  All have built-in examples of a simple Shiny application, "Old Faithful Geyser Data", that can be created by choosing "File", "New file", "Shiny Web App" and then be published using the IDE's built-in "Publish" button. 
+
+## Prerequisites
+
+* Must have `shiny` R package installed
+
+## Create Content {.numbered}
+
+### 1. Create a new Shiny Web App
```

Comment:
I would tweak this to instead create a new project with a type "shiny app". In particular having the app then at the root in the current working directly will generally make tooling a little more pleasant to interact with. It will also keep them from accidentally foot-gunning themselves with other random files/values in their current session when they start trying this

Reply:
Done.